sdks/java/io/cassandra/build.gradle

```diff
@@ -18,6 +18,7 @@
 
 apply from: project(":").file("build_rules.gradle")
 applyJavaNature(artifactId: "beam-sdks-java-io-cassandra")
+enableJavaPerformanceTesting()
```

Comment:
What does this line do? Does it enable for all build?

Reply:
This adds `integrationTest` task to cassandra io's build.gradle file. It works the same way as `applyJavaNature()` but adds integration testing related things. The way it is done now requires developers explicitly "append" the tasks to every module that needs ITs by adding `enableJavaPerformanceTesting()` in build.gradle file.

An alternative would be to include this task (along with `packageIntegrationTests` ) to `applyJavaNature()` closure. This would add the tasks (`integrationTests`, `packageIntegrationTest` to every module that uses `applyJavaNature()`, so it's more implicit. 

What do you think? Which option do you like better?

Reply:
The current approach seems better. Thank you for the explanation.